sdk/consumption/arm-consumption/package.json

```diff
@@ -107,5 +97,45 @@
     ],
     "disableDocsMs": true,
     "apiRefLink": "https://docs.microsoft.com/javascript/api/@azure/arm-consumption?view=azure-node-preview"
+  },
+  "type": "module",
+  "tshy": {
+    "project": "./tsconfig.src.json",
+    "exports": {
+      "./package.json": "./package.json",
+      ".": "./src/index.ts"
+    },
+    "dialects": [
+      "esm",
+      "commonjs"
+    ],
+    "esmDialects": [
```

Comment:
Are we shipping for React-Native and Browser support?  If not, please remove the `esmDialects`.